joplin/api/schema.py

```diff
@@ -506,12 +503,35 @@ class Meta:
         fields = ['date']
 
 
+class OfficialDocumentNodeDocument(graphene.ObjectType):
+    filename = graphene.String()
+    fileSize = graphene.String()
+
+
 class OfficialDocumentPageOfficialDocumentNode(DjangoObjectType):
+    document = graphene.Field(OfficialDocumentNodeDocument)
+
     class Meta:
         model = OfficialDocumentPageOfficialDocument
         filter_fields = ['date']
         interfaces = [graphene.Node]
 
+    def resolve_document(self, info):
+        english_doc = OfficialDocumentNodeDocument(
+            filename=self.document.filename,
+            fileSize=self.document.file_size,
+        )
+        if django.utils.translation.get_language() == 'es':
```

Comment:
A question about this - is it possible for a self.document_es to be incomplete? And some fields within a Spanish document would have to fallback to the english version?

Reply:
as in, a file name but no file_size? the file_size comes from wagtail/django and the document the user uploads

Reply:
Yeah, I guess you couldn't save a spanish document without a document. Makes sense. Cool, ship it! :shipit: